### ISSUE TITLE
Learning structure in gene expression data using deep architectures, with an application to gene clustering

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#65]
http://doi.org/10.1109/BIBM.2015.7359871

**Comment by gwaygenomics** [comment]
Really nice discussion of denoising autoencoders, stacked denoising autoencoders, and how they relate to deep belief networks. The paper stops short of evaluating stacked denoising autoencoders but shows that denoising autoencoder features cluster gene expression data with more concordance than raw or PCA'd features. 

### Biology

* Clustering yeast microarray data into known cell cycle gene modules. 

### Computational aspects

* single layer denoising autencoder
* parameter sweep with several hyperparameters
  * batch size, number of epochs, number of hidden nodes, corruption level, learning rate
* report [rand index](https://en.wikipedia.org/wiki/Rand_index "rand index wiki") as performance metric comparing raw data and DA feature data clustering (k means and spectral clustering)
  * also compare DA features to PCA features

### Results

* Better performance recapitulating known yeast cell cycle gene modules using shallow denoising autoencoder features than using raw or PCA features.

**Comment by cgreene** [comment]
Looks like it also cites @tj8901nm's #6. PCA results, though a different analysis, are concordant with findings in #22 from @tj8901nm as well. That also compared to ICA.

The results comparing against baseline are quite interesting. The data reconstructed by the NN model actually comes out looking better than baseline by their evaluation. That's something that I wouldn't necessarily expect.
